README.md

```diff
@@ -10,30 +10,30 @@ as described in the Rust API Guidelines [[1](https://rust-lang.github.io/api-gui
 
 ```toml
 [dependencies]
-sealed = "0.1"
+sealed = "0.2"
```

Comment:
This won't find `0.2.0-rc1` version. You need to keep it in sync with `Cargo.toml`.